api-tests/cdc-service/cdc-service_ecommerce_local.postman_collection.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
why duplicating this collection? you can use tests for pagopa-ecommerce-transactions-service, there are no new api to be invoked here